paddle/operators/listen_and_serv_op.cc

```diff
@@ -155,6 +158,11 @@ class ListenAndServOp : public framework::OperatorBase {
       } catch (std::exception &e) {
         LOG(ERROR) << "run sub program error " << e.what();
       }
+
+      // TODO(Yancey1989): use an operator to reset sprase variable
```

Comment:
How about add more comments about how to achieve sparse updating?

Reply:
Done, I will add some details about remote sparse update in the dist desig doc.